specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-08-01-preview/IncidentCreationRules.json

```diff
@@ -0,0 +1,640 @@
+          "format": "int32",
+          "description": "The order of evaluation of the incident creation rule",
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 1000
+        },
+        "conditions": {
+          "description": "The conditions to evaluate to determine if the incident creation rule should be applied.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IncidentCreationRuleCondition"
+          },
+          "x-ms-identifiers": []
+        },
+        "createIncident": {
+          "description": "Determines whether to create an incident based on this rule or not.",
+          "type": "boolean"
+        },
+        "groupingConfiguration": {
+          "$ref": "#/definitions/IncidentCreationRuleGroupingConfiguration"
+        },
+        "enabled": {
+          "description": "Determines whether the incident creation rule is enabled or disabled.",
+          "type": "boolean"
+        },
+        "lastModifiedTimeUtc": {
+          "format": "date-time",
+          "description": "The last time the incident creation rule was updated.",
+          "type": "string",
+          "readOnly": true
+        },
+        "createdTimeUtc": {
+          "format": "date-time",
+          "description": "The time the incident creation rule was created.",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastModifiedBy": {
+          "type": "object",
+          "$ref": "../../../common/2.0/types.json#/definitions/ClientInfo"
+        },
+        "createdBy": {
+          "type": "object",
+          "$ref": "../../../common/2.0/types.json#/definitions/ClientInfo"
+        }
+      }
+    },
+    "IncidentCreationRulePropertyBasedGroupingProperties": {
+      "type": "object",
+      "properties": {
+        "groupingAttributes": {
+          "description": "The grouping attributes that will determine if the alert should be grouped to an incident.",
+          "type": "array",
+          "maxItems": 5,
+          "items": {
+            "$ref": "#/definitions/IncidentCreationRuleGroupingAttribute"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "IncidentCreationRulePropertyConditionSupportedOperator": {
+      "enum": [
+        "Equals",
+        "NotEquals",
+        "Contains",
+        "NotContains",
+        "StartsWith",
+        "NotStartsWith",
+        "EndsWith",
+        "NotEndsWith"
+      ],
+      "type": "string",
+      "example": "Equals",
+      "x-ms-enum": {
+        "name": "IncidentCreationRulePropertyConditionSupportedOperator",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Equals",
+            "description": "Evaluates if the property equals at least one of the condition values."
+          },
+          {
+            "value": "NotEquals",
+            "description": "Evaluates if the property does not equal any of the condition values."
+          },
```

Comment:
If that's the meaning, would it be clearer to call this DoesNotEqualAny? or MatchesNone?

Reply:
I have similar thoughts about the below conditions, I wonder if I understand the usage correctly. Some good examples might help.